idaes/generic_models/properties/core/eos/ceos.py

```diff
@@ -70,10 +71,12 @@ class CubicType(Enum):
 
 
 class MixingRuleA(Enum):
+    # Rule to calculate am for cubic equations of state
     default = 0
```

Comment:
We probably should add another entry here with an explicit name for the default rule (van der Waal's?), and the machinery to link that up later. This would also give us the ability to properly test the machinery (even if the end result is the same).